app/controllers/users_controller.rb

```diff
@@ -52,7 +52,7 @@ def confirm_email
   private
 
   def set_user
```

Comment:
In this case, the benefits are not significant. However, if you have repetitive occurrences of the line @user = User.find(params[:id]), it would be advisable to utilize the ||= operator. This approach allows you to completely remove the before_action :set_user callback and instead use the user method in place of @user. In this scenario, it would be preferable to rename the method as user instead of set_user.

Your actions would appear as follows:

```ruby
def edit; end
```
To enable the usage of the user method within views, configure it as follows:

```ruby
def user
  @user ||= User.find(params[:id])
end
helper_method: user
```
By making these adjustments, you can eliminate repetition and enhance the readability of your code.

Reply:
@altivi, good idea! I I didn't even think about it xD